site/en/docs/extensions/mv3/favicon/index.md

```diff
@@ -27,7 +27,7 @@ chrome-extension://EXTENSION_ID/_favicon/?pageUrl=EXAMPLE_URL&size=FAV_SIZE
 : The URL of the favicon's website.
 
 `FAV_SIZE`
-: The size of the favicon. For example: 16, 32, or 64.
+: The size of the favicon. The most common size is 16 pixels.
```

Comment:
I agree with adding your line. Should we not list what the common sizes are? Is that information somewhere else?

Reply:
I'm going by [MDN's favicon doc](https://developer.mozilla.org/en-US/docs/Glossary/Favicon) which mentions the most common size.

Reply:
I'm not objecting to adding that. I'm asking if you should say anything about other sizes.

Reply:
I defer to your judgment on this matter. Please make a suggestion.

Reply:
I misread this. This is 'favorite size' not 'icon sizes'. That's why I was asking for more.